packages/eslint-plugin/docs/rules/no-implicit-any-catch.md

```diff
@@ -8,6 +8,8 @@ By default, TypeScript will type a catch clause variable as `any`, so explicitly
 
 The `noImplicitAny` flag in TypeScript does not cover this for backwards compatibility reasons, however you can use `useUnknownInCatchVariables` (part of `strict`) instead of this rule.
 
+## DEPRECATED
```

Comment:
Hi! I noticed that this has been deprecated, but this seems like an unrelated rule to `no-duplicate-imports`

Is this intentional? If not, is there a different rule we should use?

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/186fcf7bdee8d4cb33bc2f26b2961529f1bae493/packages/eslint-plugin/src/rules/no-implicit-any-catch.ts#L17

It was added [here](https://github.com/typescript-eslint/typescript-eslint/commit/deeb7bb9334d301c6af56aefd37d318231af11ef)

> [TS 4.4 now has a compiler flag for this](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4-rc/#use-unknown-catch-variables). So it's now a rule we don't need! Because 4.4 is still in RC, we'll just deprecate it, and remove it in the next major.